docs/reference/contributing/guidelines/pull_request_drafts/pr_breaking_change.md

```diff
@@ -0,0 +1,11 @@
+<!--
+    These sections should part of the PR template for any breaking change
+-->
+
+### Release notes
+
+#### Description
+
+#### Analysis of effects
+
+#### Migration
```

Comment:
We already have a description section at the top of PRs I think having another one under Release Notes would be confusing. I would go more with something like:

#### Reason for change

#### Impact of change

#### Migration path

Reply:
fixed in 65d6a27